src/comm/client/client-util.js

```diff
@@ -76,6 +159,10 @@ function launchClient(updates, results) {
         else if(msg.type === 'txUpdated') {
             pushUpdate(pid, msg.data);
         }
+        else if(msg.type === 'txUpdatedWithMQ') {
+            global_pid = pid;
```

Comment:
Does this global_pid affect the latency computation?

Reply:
Nope. its not the issue of global_pid